.envrc

```diff
@@ -5,17 +5,11 @@ missing() {
 
 set -e
 
-if missing pre-commit; then
-    echo >&2 "You're missing pre-commit. Install it with brew install pre-commit"
-    return 1
-fi
-# If installed, it executes very fast
-pre-commit install >/dev/null
 # Convenient if you want secrets to load (e.g. personal token, Github webhook secret) or override values from env.development
 dotenv_if_exists .env
 
 if [ ! -d .venv ]; then
-    python3 -m venv .venv
+    python3.11 -m venv .venv
```

Comment:
this was fine before -- it'll use the correct python due to either pyenv or `devenv`